src/Symfony/Bridge/Doctrine/.gitattributes

```diff
@@ -0,0 +1,2 @@
+/Tests export-ignore
```

Comment:
you should also add ``.git* export-ignore`` to ignore the gitattributes and gitignore from the archive